utils/leagueHelpers.ts

```diff
@@ -5,3 +5,10 @@ export const leagueNameToId = (league: League) => leagues.indexOf(league);
 
 export const isMainLeague = (league: League): boolean =>
   league === 'shl' || league === 'smjhl';
+
+export const shouldShowDivision = (league: League, season: number): boolean => { 
+  if (league === 'smjhl' && season >= 76) {
+    return false;
+  }
+  return true;
+};
```

Comment:
Let's encapsulate the isMainLeague check into here so we only have to call this when determining whether to show divisions. This will read better than the two split apart because we only care if we should show the division over if we are on a "Main League".

```suggestion
export const shouldShowDivision = (league: League, season: number): boolean => { 
  if (isMainLeague(league) || (league === 'smjhl' && season >= 76)) {
    return false;
  }
  return true;
};
```

Reply:
never thought about it like that. great suggestion.

Reply:
actually, this edit will not work as with IIHF & WJC using && will include division tab. If we use || division will get removed from SHL as well. 

`
export const shouldShowDivision = (league: League, season: number): boolean => { 
  if (!isMainLeague(league)) {
    return false;
  }
  if (league === 'smjhl' && season >= 76) {
    return false;
  }
  return true;
};
`

would make it work

Reply:
Ah my bad yeah it should be

```suggestion
export const shouldShowDivision = (league: League, season: number): boolean => { 
  if (!isMainLeague(league) || (league === 'smjhl' && season >= 76)) {
    return false;
  }
  return true;
```


OR 


```suggestion
export const shouldShowDivision = (league: League, season: number): boolean => { 
  return league === 'shl' || (league === 'smjhl' && season < 76);
};
```
which i think honestly reads the best

Reply:
I like the 2nd expression the best as well